src/base/net/portforwarder.cpp

```diff
@@ -28,110 +28,21 @@
 
 #include "portforwarder.h"
 
-#include <QDebug>
-
-#include <libtorrent/session.hpp>
-
-#include "base/logger.h"
-#include "base/settingsstorage.h"
-
-static const QString KEY_ENABLED = QStringLiteral("Network/PortForwardingEnabled");
-
-namespace libt = libtorrent;
-using namespace Net;
-
-PortForwarder::PortForwarder(libtorrent::session *provider, QObject *parent)
-    : QObject(parent)
-    , m_active(false)
-    , m_provider(provider)
-{
-    if (SettingsStorage::instance()->loadValue(KEY_ENABLED, true).toBool())
-        start();
-}
-
-PortForwarder::~PortForwarder()
+Net::PortForwarder::PortForwarder(QObject *parent)
```

Comment:
can you explain why this refactor?

Reply:
>can you explain why this refactor?

Do you mean using namespace explicitly? Or overall refactor of port forwarder?

Reply:
>Or overall refactor of port forwarder?

This.

Reply:
Actually this is a part of more global refactoring. This class just came to hand first.

I want to separate the abstract interface from the implementation in all those classes that are implemented using the libtorrent. Currently their headers contain some implementation details that should not concern external